examples/pion-to-pion/offer/main.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 func main() {
-	addr := flag.String("address", ":50000", "Address that the HTTP server is hosted on.")
+	addr := flag.String("address", "127.0.0.1:50000", "Address that the HTTP server is hosted on.")
```

Comment:
Mind leaving it as `:50000`? Beyond that PR looks perfect to me! Thank you so much for the contribution :)

Reply:
I could, but then the `fetch()` will still fail if this is run in a browser.
Technically I guess its a problem with `http.Post` but it might be worth working around here I think.

Thoughts?

Reply:
Aye! That makes sense to me, will merge now :)

Thanks for fixing